gcc/rust/typecheck/rust-hir-type-check-item.h

```diff
@@ -31,14 +31,16 @@ namespace Resolver {
 
 class TypeCheckItem : public TypeCheckBase
 {
+  using TypeCheckBase::visit;
```

Comment:
Does this line actually need to be here? `TypeCheckBase::visit` doesn't seem to be referenced in the class anywhere.

Reply:
good find i was testing something out to do with overloaded virtual and it didnt work